index.html

```diff
@@ -48,6 +48,229 @@ <h1 class="logo-text">Pascal</h1>
+                </li>
+                <li class="data for-js">
+                    <p>JavaSript</p>
+                </li>
+            </ul>
+            <div class="see-project">
+                <a href="#" class="see">See Project</a>
+            </div>
+
+        </div>
+        
+        <div class="card">
+            <div class="card-header2">
+            </div>
+            <h2 class="card-titles">Multi-Post
+                Stories</h2>
+            <ul class="bandle">
+                <li>
+                    <p class="canopy">CANOPY</p>
+                </li>
+                <li><img src="./img/cannopy.png" class="canopy-circle" alt="canopy" /></li>
+                <li>
+                    <p class="role">Backend Dev</p>
+                </li>
+                <li><img src="./img/cannopy.png" class="canopy-circle" alt="canopy" /></li>
+                <li>
+                    <p class="role">2015</p>
+                </li>
+            </ul>
+
+            <div class="card-content">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <ul class="content_btn">
+                <li class="data for-html">
+                    <p>html</p>
+                </li>
+                <li class="data for-css">
+                    <p>CSS</p>
+                </li>
+                <li class="data for-js">
+                    <p>JavaSript</p>
+                </li>
+            </ul>
+            <div class="see-project">
+                <a href="#" class="see">See Project</a>
+            </div>
+
+        </div>
+
+        
+        <div class="card">
+            <div class="card-header1">
+            </div>
+            <h2 class="card-titles">Multi-Post Stories</h2>
+
+
+            <ul class="bandle">
+                <li>
+                    <p class="canopy">CANOPY</p>
+                </li>
+                <li><img src="./img/cannopy.png" class="canopy-circle" alt="canopy" /></li>
+                <li>
+                    <p class="role">Backend Dev</p>
+                </li>
+                <li><img src="./img/cannopy.png" class="canopy-circle" alt="canopy" /></li>
+                <li>
+                    <p class="role">2015</p>
+                </li>
+            </ul>
+
+            <div class="card-content">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <ul class="content_btn">
+                <li class="data for-html">
+                    <p>html</p>
+                </li>
+                <li class="data for-css">
+                    <p>CSS</p>
+                </li>
+                <li class="data for-js">
+                    <p>JavaSript</p>
+                </li>
+            </ul>
+            <div class="see-project">
+                <a href="#" class="see">See Project</a>
+            </div>
```

Comment:
[Optional Changes] These changes won't stop your project from being approved but it would be nice you consider them as they will help improve your project and your skills as a professional developer that writes good code following HTML5 standards.

- [ ] It would be nice you make this a `button`  tag and not a `div` tag. Eg. 
`<button class="see-project">
            Get my resume
        </button>`
OR 
You leave it as an `a` tag - `<a href="#" class="see">See Project</a>` then pass your class to that `a` tag and style it

- [ ] Also add hover and active state on every button you have in the project as this was required for you to implement.